subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -442,73 +442,67 @@ static void wifi_mgmt_event_handler(struct net_mgmt_event_callback *cb,
 	}
```

Comment:
Please change the commit subject to `net: wifi: shell: ...` for both commits.

Also add information in the commit message why this change is needed, it might make sense to add information about this change to 3.7 migration guide as it changes user seeing interface.

Reply:
Done